### PR TITLE
Plot NAS quotas

### PR DESCRIPTION
**Pull request by kate-v-stepanova**
Analog of UPPMAX quotas page, but for NASes disk usage:
https://genomics-status-stage.scilifelab.se/nas_quotas

### CODE REVIEW

File: run_dir/static/js/nas_quotas.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  data = JSON.stringify(data);
+  data = JSON.parse(data);
+  data = JSON.parse(data); // one more time, i don't know why but otherwise it doesn't work
+  return data;
+}
+
+function plot_chart(nas_url, plot_data, limit_data, max_x_value, nas_id) {
+    // Get timestamp for 2 months ago
+    var d = new Date();
+    d.setMonth(d.getMonth() - 2);
```
Comment (Galithil) [inline run_dir/static/js/nas_quotas.js]: what happens here in january ?

Comment (kate-v-stepanova) [inline run_dir/static/js/nas_quotas.js]: In January it will show November last year

**Comment by kate-v-stepanova** [comment]
Can this one be merged?

**Comment by kate-v-stepanova** [comment]
ahh I pushed to the wrong branch, so this PR was overwritten by another change

**Comment by ewels** [comment]
I missed it sorry. Can we see it on stage?

**Comment by kate-v-stepanova** [comment]
well.. I made a mistake with NAS Quotas (git push --force), and now I have to recover it from the history. But the quotas page is deployed on stage. (the problem with plots we talked about is fixed)

**Comment by kate-v-stepanova** [comment]
And don't close this PR now please, I will try to recover the previous changes first

**Pull request merged**